lib/rummager.rb

```diff
@@ -3,13 +3,14 @@
 require 'active_support/core_ext/array'
 require 'active_support/core_ext/hash'
 require 'active_support/core_ext/string'
-require 'airbrake'
 require 'cgi'
 require 'csv'
 require 'elasticsearch'
 require 'erb'
 require 'fileutils'
 require 'gds_api/publishing_api_v2'
+require 'sentry-raven'
```

Comment:
@tijmenb: Do you think [configure.rb in govuk_app_config](https://github.com/alphagov/govuk_app_config/blob/master/lib/govuk_app_config/configure.rb) should `require sentry-raven`? This works, but we were a bit surprised that we had to add it (and make sure that it gets required before govuk_app_config). Though I realise we have to do a bit of extra work because rummager isn't on Rails. 🙂

Reply:
I think it already does, but in a roundabout way:

https://github.com/alphagov/govuk_app_config/blob/master/lib/govuk_app_config.rb#L3
->
https://github.com/alphagov/govuk_app_config/blob/master/lib/govuk_app_config/govuk_error.rb#L1

We should probably require it directly in https://github.com/alphagov/govuk_app_config/blob/master/lib/govuk_app_config.rb, but this still should've worked?

Reply:
Cheers. I'm going to merge this PR now so that we can do some more testing before the ES 2.4 upgrade, but I'll take a look in case we misunderstood something yesterday.